Civi/Funding/EventSubscriber/FundingCase/FundingCasePermissionsCacheClearSubscriber.php

```diff
@@ -79,12 +84,67 @@ public function preContact(PreEvent $event): void {
     }
   }
 
+  /**
+   * @throws \CRM_Core_Exception
+   */
+  public function onPreGroup(PreEvent $event): void {
+    if (NULL === $event->id) {
+      return;
+    }
+
+    if ('delete' !== $event->action) {
+      $changeSet = $this->changeSetFactory->createChangeSetForPreEvent($event, ['is_active']);
+      if (!isset($changeSet['is_active'])) {
+        return;
+      }
+    }
+
+    foreach ($this->getRelationshipRelationsByGroupId($event->id) as $relation) {
+      $this->permissionsCacheManager->clearByFundingCaseId($relation->getFundingCaseId());
+    }
+  }
+
+  /**
+   * @throws \CRM_Core_Exception
```

Comment:
perhaps add a short description why permission-cache gets cleared for the related funding-cases of a group contact?
what is the effect of clearing?